chart/templates/terraform_controller.yaml

```diff
@@ -20,3 +22,9 @@ spec:
         - name: terraform-controller
           image: "{{ .Values.image.name }}"
           imagePullPolicy: {{ .Values.image.imagePullPolicy }}
+          env:
+            - name: CONTROLLER_NAMESPACE
```

Comment:
Great. It's so brilliant to let the controller get the release namespace!

Reply:
why it's needed in controller?

Reply:
So that the controller can provision resources in the namespace it's deployed in (e.g. I've been deploying it in a namespace called `tf-controller`, so the batch jobs, config maps, etc are deployed in `tf-controller` too)